webapp/src/components/post_type_webex/post_type_webex.jsx

```diff
@@ -0,0 +1,220 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {makeStyleFromTheme} from 'mattermost-redux/utils/theme_utils';
+
+export default class PostTypeWebex extends React.PureComponent {
+    static propTypes = {
+
+        /*
+         * The post to render the message for.
+         */
+        post: PropTypes.object.isRequired,
+
+        /**
+         * Set to render post body compactly.
+         */
+        compactDisplay: PropTypes.bool,
+
+        /**
+         * Flags if the post_message_view is for the RHS (Reply).
+         */
+        isRHS: PropTypes.bool,
+
+        /**
+         * Set to display times using 24 hours.
+         */
+        useMilitaryTime: PropTypes.bool,
+
+        /*
+         * Logged in user's theme.
+         */
+        theme: PropTypes.object.isRequired,
+
+        /*
+         * Creator's name.
+         */
+        creatorName: PropTypes.string.isRequired,
+    };
+
+    static defaultProps = {
+        mentionKeys: [],
+        compactDisplay: false,
+        isRHS: false,
+    };
+
+    render() {
+        const style = getStyle(this.props.theme);
+        const post = this.props.post;
+        const props = post.props || {};
+
+        let preText;
+        let content;
+        let subtitle;
+        if (props.meeting_status === 'STARTED') {
+            preText = `${this.props.creatorName} has started a meeting`;
+            content = (
+                <a
+                    className='btn btn-lg btn-primary'
+                    style={style.button}
+                    rel='noopener noreferrer'
+                    target='_blank'
+                    href={props.meeting_link}
+                >
+                    <i
+                        style={style.buttonIcon}
+                    >
+                        <svg
```

Comment:
makes sense to move it to the Svgs module? maybe we replace the current data structure and export functions components that renders the Svgs, and maybe in this cases accepts a "sytle" prop. It is just a possible improvement, If you don't see it or see it over-complication, feel free to discard this comment.